Makefile

```diff
@@ -5,10 +5,11 @@ MM_DEBUG ?=
 GOPATH ?= $(shell go env GOPATH)
 GO_TEST_FLAGS ?= -race
 GO_BUILD_FLAGS ?=
+GO_BUILD_TAGS ?=
 MM_UTILITIES_DIR ?= ../mattermost-utilities
 DLV_DEBUG_PORT := 2346
-DEFAULT_GOOS := $(shell go env GOOS)
-DEFAULT_GOARCH := $(shell go env GOARCH)
+DEFAULT_GOOS ?= $(shell go env GOOS)
+DEFAULT_GOARCH ?= $(shell go env GOARCH)
```

Comment:
Allow `ce2e/` to override the "default" GOOS so we can explicitly build for `linux` and `amd64`, matching the Docker containers.